Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -288,9 +300,21 @@ extension Docc {
                     }
 
                 } else {
-                    throw TemplateOption.missingHTMLTemplateError(
-                        path: templateOption.defaultTemplateURL.path
+                    let missingTemplateDiagnostic = Diagnostic(
```

Comment:
Should we move this into a static function of `TemplateOption`?
> `TemplateOption.invalidHTMLTemplateError` and `TemplateOption.missingHTMLTemplateError`

Reply:
Also the variable name `missingTemplateDiagnostic` seems strange since the summary and explanation says "Invalid or missing"

Reply:
> Should we move this into a static function of TemplateOption?
>
> TemplateOption.invalidHTMLTemplateError and TemplateOption.missingHTMLTemplateError

We currently only have the one need for an invalid template warning (non-error) so I think this is okay for now. We can refactor this if the need comes up in the future.

Reply:
> Also the variable name missingTemplateDiagnostic seems strange since the summary and explanation says "Invalid or missing"

That's fair! I updated it to be `invalidOrMissingTemplateDiagnostic`.